discord/ext/tasks/__init__.py

```diff
@@ -166,7 +173,11 @@ async def _loop(self, *args: Any, **kwargs: Any) -> None:
                     self._last_iteration = self._next_iteration
                     self._next_iteration = self._get_next_sleep_time()
                 try:
-                    await self.coro(*args, **kwargs)
+                    if self.overlap:
+                        task = asyncio.create_task(self.coro(*args, **kwargs))
+                        self._tasks.append(task)
```

Comment:
redundant variable
also can we set a name here? maybe `pycord-loop-<i>`?
```suggestion
                        self._tasks.append(asyncio.create_task(self.coro(*args, **kwargs), name=...))
```

Reply:
changed with : 
```py
self._tasks.append(
        asyncio.create_task(
            self.coro(*args, **kwargs),
            name=f"pycord-loop-{self.coro.__name__}-{self._current_loop}",
        )
    )
```